lm_eval/models/gpt2.py

```diff
@@ -59,10 +68,29 @@ def __init__(
         # multithreading and batching
         self.batch_size_per_gpu = batch_size  # todo: adaptive batch size
 
-        # TODO: fix multi-gpu
-        # gpus = torch.cuda.device_count()
-        # if gpus > 1:
-        #     self.gpt2 = nn.DataParallel(self.gpt2)
+        # multigpu support with accelerate
+        if gpus > 1:
+            accelerator = Accelerator(device_placement=False)
```

Comment:
I was able to run with `Accelerator()` is there a specific configuration that requires `device_placement=False`? I'm not as familiar as I should be with accelerate.